index.bs

```diff
@@ -891,16 +901,14 @@ The {{MLActivation}} objects (including the ones passed as input to methods) are
     To <dfn>create an MLActivation</dfn> given |builder|, |name|, |options| and |init-steps|, run the following steps:
   </summary>
   <div class=algorithm-steps>
-    1. [=Assert=]: the type of |builder| is {{MLGraphBuilder}}.
-    1. If |name| is empty, then [=exception/throw=] a "{{TypeError}}".
     1. Let |activation| be a new [=object=].
     1. Set |activation|.{{MLActivation/[[builder]]}} to |builder|.
     1. Set |activation|.{{MLActivation/[[name]]}} to |name|.
     1. If |options| is an [=object=], set |activation|.{{MLActivation/[[options]]}} to |options|.
     1. If any of the following sub-steps fail, [=exception/throw=] an "{{OperationError}}" {{DOMException}}.
         1. Make a request to the underlying platform to:
             1. Create an [=implementation-defined=] platform operator |opImpl| for the given |name| operation.
-            1. Store a reference of |opImpl| in |activation|.{{MLActivation/[[operator]]}}.
+            1. Set |activation|.{{MLActivation/[[operator]]}} to |opImpl|.
```

Comment:
@huningxin do we want to convey more here (and similar places) than just setting the internal slot?

Reply:
Just setting the internal slot might be fine. I suppose the graph build algorithm (to be specified) would fetch the platform operator from the internal slot at the graph building stage.